Cargo.toml

```diff
@@ -43,6 +43,12 @@ inotify = "^0.1"
 [target.x86_64-unknown-linux-musl.dependencies]
 inotify = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+version = "^0.5"
+
+[target.x86_64-unknown-linux-musl.dependencies.mio]
+version = "^0.5"
```

Comment:
Could you use the style as shown by other stanzas? Add `mio = "^0.5"` to the relevant stanzas.